server/protocols/jmap/src/test/resources/calendar.eml

```diff
@@ -0,0 +1,42 @@
+Return-Path: <local@localhost.com>
+MIME-Version: 1.0
+Delivered-To: to@localhost.com
+From: User From <fromUser@localhost.com>
+Message-ID: <2d0fa16c-4c37-58de-1acc-50b5f6a23a60@localhost.com>
+To: User To <toUser@linagora.com>
+Date: Wed, 26 Jul 2017 10:34:02 +0200
+Subject: Any subject
+Content-class: urn:content-classes:calendarmessage
```

Comment:
Looks like the Content-class is related to the text/calendar part.

IMO, after seeing this, **Content-\*** headers should be copied to the mime body part text/calendar and removed from the top level headers...